feconf.py

```diff
@@ -650,19 +650,27 @@ def get_empty_ratings():
 MAX_PLAYTHROUGHS_FOR_ISSUE = 5
 # Probability of recording a playthrough.
 RECORD_PLAYTHROUGH_PROBABILITY = 0.2
-# IDs of allowed types of issues.
-ALLOWED_ISSUE_IDS = [
+# Types of allowed issues.
+ALLOWED_ISSUE_TYPES = [
     'EarlyQuit',
     'MultipleIncorrectSubmissions',
     'CyclicStateTransitions'
 ]
-# IDs of allowed learner actions.
-ALLOWED_ACTION_IDS = [
+# Types of allowed learner actions.
+ALLOWED_ACTION_TYPES = [
     'ExplorationStart',
     'AnswerSubmit',
     'ExplorationQuit'
 ]
-
+# Mapping from issue type to issue keyname in the issue customization dict. This
+# particular mapping is to distinguish between issues which have their keyname
```

Comment:
This second sentence isn't great because new types of issues may be added in the future.

Instead, say something about how the field can be used to uniquely identify/characterize the issue.